services/multiplayer/MultiplayerManager.gd

```diff
@@ -0,0 +1,90 @@
+extends Node
+
+# Maybe we can export theses constants to a global variables project ?
+const DEFAULT_PORT = 28000
+const MAX_PEERS = 4
+
+var peer: ENetMultiplayerPeer
+var player_name := "Pirate"
+
+# Names for remote players in id:name format.
+var players := {}
+
+# Signals to let lobby GUI know what's going on.
+signal player_list_changed()
+signal connection_failed()
+signal connection_succeeded()
+signal game_error(error: int)
```

Comment:
Did the comment taught us something?

Reply:
I don't think it's a relevant information, the signals names already give us the information we need.